docs/api/index.rst

```diff
@@ -11,6 +11,13 @@ API
 Changelog
 ---------
 
+1.11 (unreleased)
+'''''''''''''''''
+
+- The ``permissions`` are now empty in the response if the user has not the
```

Comment:
somehow this doesn't sound very good... maybe reword to "if the user does not have the permission to write"?